webClient/src/app/app.component.ts

```diff
@@ -11,7 +11,7 @@
 */
 
 import { Component, Inject } from '@angular/core';
-
+import { LanguageLocaleService } from '../../../../zlux-app-manager/virtual-desktop/src/app/i18n/language-locale.service';
```

Comment:
Hopefully this isnt needed directly but if it is, think of another way because referencing stuff from far within another repo seems error prone here.
I am imagining that ZoweZLUX.globalization has the info you need via

```typescript
 interface Globalization {
    getLanguage(): string;
    getLocale(): string;
  }
```